app/(pages)/explore/[slug]/page.tsx

```
@@ -72,6 +69,12 @@ const ExplorePage = () => {
       ) : (
         <div className=" container">
           <div className="flex items-center gap-x-[28px]">
+          <button
+            className=" -translate-y-1/2 max-md:h-auto bg-[#0C001F] bg-opacity-80 px-4 rounded-md "
+            onClick={() =>  window.location.href = '/explore'}
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)